modules/ROOT/pages/bolt-compatibility.adoc

```diff
@@ -135,8 +157,10 @@
 | `(x)`
 | `x`
 |
+|
+|
 
-| `5.0` - `5.3`
+| `5.0` - `5.6`
```

Comment:
This is going to get outdated every month. Can we put `5.x` instead? There shouldn't be any breaking changes anyway, right?

Reply:
yeah i suppose we could untill we introduce a breaking change of some description (if we do)